plugins/modules/cloud/lxd/lxd_virtual_machine.py

```diff
@@ -0,0 +1,744 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2016, Hiroaki Nakamura <hnakamur@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = '''
+---
+module: lxd_virtual_machine
+short_description: Manage LXD Virtual machines
+description:
+  - Management of LXD virtual machines
+author: "Hiroaki Nakamura (@hnakamur)"
+options:
+    name:
+        description:
+          - Name of a virtual machine.
+        type: str
+        required: true
+    architecture:
+        description:
+          - The architecture for the virtual machine (e.g. "x86_64" or "i686").
+            See U(https://github.com/lxc/lxd/blob/master/doc/rest-api.md#post-1)
+        type: str
+        required: false
+    config:
+        description:
+          - 'The config for the virtual machine (e.g. {"limits.cpu": "2"}).
```

Comment:
```suggestion
          - 'The config for the virtual machine (for example C({"limits.cpu": "2"})).
```
Avoid Latin abbrevations. Also see https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#linking-and-other-format-macros-within-module-documentation for more on formatting.